create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+
+```bash
+brew info doctl
+brew install doctl
+```
+
+After installation we have to initialize it using the DigitalOcean personal token created at the beginning of this blueprint (when asked just paste it):
+
+```bash
+doctl auth init
+```
+
+Verify that the new auth context was added with:
+
+```bash
+doctl auth list
+```
+
+### Kubectl
+
+Please follow the specific steps for your OS and architecture from the official [site](https://kubernetes.io/docs/tasks/tools). On MacOS we can use Homebrew to install it very easily like this:
+
+```bash
+brew info kubectl
+brew install kubectl
+```
+
+Next we have to set kubectl context to point to our cluster created in this blueprint like this:
+
+```bash
+doctl k8s cluster kubeconfig save <doks_cluster_name>
+```
+
+Where `<doks_cluster_name>` must be replaced with the name given to the cluster in the `project.tfvars` file created in this blueprint (the variable is `doks_cluster_name`).
+
+Next check that the context was set and it's pointing to your Kubernetes cluster by running:
+
+```bash
+kubectl config get-contexts
+```
+
+### Flux
+
+This is the CLI tool used by Flux to provision it as well as interact with it. Can be installed from the [official](https://fluxcd.io/docs/installation) site following the steps specific for your OS.
+
+After installation and if all the above steps for doctl and kubectl were completed successfully we can check flux status with:
+
+```bash
+flux check
+```
+Sample output:
+
+```
+► checking prerequisites
+✔ kubectl 1.21.3 >=1.18.0-0
+✔ Kubernetes 1.21.2 >=1.16.0-0
+► checking controllers
+✗ helm-controller: deployment not ready
+► ghcr.io/fluxcd/helm-controller:v0.11.1
+✔ kustomize-controller: deployment ready
+► ghcr.io/fluxcd/kustomize-controller:v0.13.1
+✔ notification-controller: deployment ready
+► ghcr.io/fluxcd/notification-controller:v0.15.0
+✔ source-controller: deployment ready
+► ghcr.io/fluxcd/source-controller:v0.15.3
+✔ all checks passed
+```
+
+Inspect all resources with:
+
+```bash
+flux get all
+```
+
+Sample output:
+
+```
+NAME                     	READY	MESSAGE                                                        	REVISION                                     	SUSPENDED 
+gitrepository/flux-system	True 	Fetched revision: main/1d69c3c9591e18a68d5794c2d21d4b31c3398ac6	main/1d69c3c9591e18a68d5794c2d21d4b31c3398ac6	False    	
+
+NAME                     	READY	MESSAGE                                                        	REVISION                                     	SUSPENDED 
+kustomization/flux-system	True 	Applied revision: main/1d69c3c9591e18a68d5794c2d21d4b31c3398ac6	main/1d69c3c9591e18a68d5794c2d21d4b31c3398ac6	False  
+```
+
+In case we need to do some troubleshooting and also see what Flux CD is doing we can access the logs via:
+
+```bash
+flux logs
+```
+
+Sample output:
```

Comment:
```suggestion
The output looks similar to the following:
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.